src/libs/ValidationUtils.js

```diff
@@ -206,7 +206,7 @@ function isValidURL(url) {
  * @returns {Object}
  */
 function validateIdentity(identity) {
-    const requiredFields = ['firstName', 'lastName', 'street', 'city', 'zipCode', 'state', 'ssnLast4', 'dob'];
+    const requiredFields = ['firstName', 'lastName', 'addressStreet', 'addressCity', 'addressZipCode', 'addressState', 'ssnLast4', 'dob'];
```

Comment:
Changed `street`, `city`, `zipCode`, `state` to `addressStreet`, `addressCity`, `addressZipCode`, `addressState` because `IdentityForm` was using both ways of referring to these inputs.

Reply:
Not sure I follow the reasoning. Did it not work before? What is the advantage of this change?

Reply:
I did it for consistency, to avoid having something like this:

https://github.com/Expensify/App/blob/2aaa8fd10bec0cfb6346140595a5f55540edacd9/src/pages/ReimbursementAccount/IdentityForm.js#L141-L142

Where we feed the input with `city`, but then we call `onFieldChange('addressCity', value)`.